snuba/settings.py

```diff
@@ -162,4 +167,28 @@ def _load_settings(obj: MutableMapping[str, Any] = locals()) -> None:
                 obj[attr] = getattr(settings_module, attr)
 
 
+# Rudimentary validation function
+def _validate_settings() -> None:
+    logger = logging.getLogger("snuba.settings")
+
+    if QUERIES_TOPIC != "snuba-queries":
```

Comment:
It doesn't seem like `QUERIES_TOPIC` is being used anymore. Can we just unilaterally do that? Won't that make this a breaking upgrade if someone has overridden `QUERIES_TOPIC` in their config?

Reply:
I think so (or at least that's not unlike what we've done to deprecate settings in the past). I think it's unlikely that someone changed this value (we didn't anywhere). This check is intended so that if someone did change the QUERIES_TOPIC value, Snuba won't start until they fix it.